capstone/src/main/resources/static/script.js

```diff
@@ -425,6 +438,31 @@ function createCalendar() {
     dateDiv.appendChild(dateDisplay);
     calendar.append(dateDiv);
   }
+}
+/*
+ * If element is selected, border will change from white to green
+ * @param element id
+ */
+function toggleBorderSelection(elementId) {
```

Comment:
Optional: this could be simplified to *only* add/remove `selected` class. Then you don't need special treatment for button.

Reply:
Oh that's how I originally had it but I realized if a user selects another choice within select, then it will remove the class even though it was still "selected"